pkg/protocol/engine/engine.go

```diff
@@ -307,6 +303,8 @@ func (e *Engine) ImportContents(reader io.ReadSeeker) (err error) {
 		return errors.Wrap(err, "failed to import commitments")
 	} else if err = e.Ledger.Import(reader); err != nil {
 		return errors.Wrap(err, "failed to import ledger")
+	} else if err := e.SybilProtection.Import(reader); err != nil {
+		return errors.Wrap(err, "failed to import epoch gadget")
```

Comment:
Can you please use `ierrors` here instead?

Reply:
Only for the added line, there rest should be replaced by my following PR anyway.